src/firefly/js/visualize/ui/MultiItemViewerView.jsx

```diff
@@ -283,7 +283,7 @@ function computeTitleYOffset(sparseGridTitleLocation, itemPosTitle,rowIdx) {
     return titleOffset;
 }
 function computeDataYOffset(sparseGridTitleLocation, itemPosTitle,rowIdx) {
-    let titleOffset= sparseGridTitleLocation==='left' && itemPosTitle[0].hasData ? 0 : 30;
+    let titleOffset= sparseGridTitleLocation==='left' && itemPosTitle[0] && itemPosTitle[0].hasData ? 0 : 30;
```

Comment:
Why 'hasData' can't also check if undefined or not? Then no need to have both in the boolean evaluation, right?

Reply:
@loi 
Can you explain the advantage of doing that way?  I know that I can do that and Emmanuel and also talked about it.  But I think using Map is  the same and have less hard coded key names.

Reply:
`get` is a lodash function.  It will handle `null` cases as well as defaults.  It's especially useful when you're fetching a deep object.  I am suggesting it in case you're not aware.  If you are and prefer your way, it's fine.  That's why it was a suggestion.

in your example, 
```
itemPosTitle[0] && itemPosTitle[0].hasData
```
it's only 2 level deep.  imagine 4 levels.. then it would look like this...
```
itemPosTitle[0] && itemPosTitle[0].main && itemPosTitle[0].main[0] && itemPosTitle[0].main[0].hasData
```
in this case, you can replace it with:
```
get(itemPosTitle, '0.main.0.hasData')
```
or, 
```
get(itemPosTitle, '0.main.0.hasData', false)
```
if you wanted the returned value to be a boolean type.